src/content/en/updates/2017/01/css-grid.md

```diff
@@ -0,0 +1,225 @@
+</video>
+
+## Placing the items
+
+One of the most powerful features of Grid is to be able to place items without
+any regards to the DOM order (although, because screen readers navigate the DOM,
+we highly recommend to be mindful about reordering elements to be properly
+accessible). If no manual placement is done, the elements are placed in the Grid
+in DOM order, arranged left to right and top to bottom. Each element occupying
+one *cell*. The order in which the grid is filled can be changed using
+[`grid-auto-flow`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow).
+
+So, how do we place elements? Arguably the easiest way to place grid items is by
+defining which columns and rows they cover. Grid offers two syntaxes to do this:
+In the first syntax you define start and end point. In the second one you define
+a start point and a span:
+
+    header {
+      grid-column: 1 / 3;
+    }
+    nav {
+      grid-row: 2 / span 2;
+    }
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/manualplacement.png">
+
+We want our header to start in the first column and end *before* the 3rd column.
+Our nav should start in the second row and span 2 rows in total.
+
+Technically, we are done implementing our layout, but I want to show you a few
+convenience features Grid provides for you to make placement easier. The first
+feature is that you can name your track borders and use those names for
+placement:
+
+    body {
+      display: grid;
+      grid-template-rows: 150px [nav-start] auto 100px [nav-end];
+      grid-template-columns: [header-start] minmax(200px, 3fr) 9fr [header-end];
+    }
+    header {
+      grid-column: header-start / header-end;
+    }
+    nav {
+      grid-row: nav-start / nav-end;
+    }
+
+The code above will yield the same layout as the code before.
+
+Even more powerful is the feature of naming entire regions in your grid:
+
+    body {
+      display: grid;
+      grid-template-rows: 150px auto 100px;
+      grid-template-columns: minmax(200px, 3fr) 9fr;
+      grid-template-areas: "header header"
+                          "nav    content"
+                          "nav    footer";
+    }
+    header {
+      grid-area: header;
+    }
+    nav {
+      grid-area: nav;
+    }
+
+`grid-template-areas` takes a string of space-separated names, allowing the
+developer to give each cell a name. If two adjacent cells have the same name,
+they are going to be coalesced to the same area. This way you can provide more
+semantics to your layout code and make media queries more intuitive. Again, this
+code will generate the same layout as before.
+
+## Is there more?
+
+Yes, yes there is, way too much to cover in a single blog post, Grid is very
+powerful. [Rachel Andrew](https://twitter.com/Rachelandrew), who is also a
+[GDE](https://developers.google.com/experts/people/rachel-andrew), is an Invited
```

Comment:
For d.g.c links, use absolute paths, not fqdn's.
`[GDE](https://developers.google.com/experts/people/rachel-andrew)` should be
`[GDE](/experts/people/rachel-andrew)`